tests/one_taskset.rs

```diff
@@ -153,3 +155,96 @@ fn test_single_taskset_empty_config_host() {
             == about.times_called()
     );
 }
+
+#[test]
+// Load test with a single task set containing two weighted tasks setup via closure.
+// Validate weighting and statistics.
+fn test_single_taskset_closure() {
+    let server = MockServer::start();
+
+    let index = Mock::new()
+        .expect_method(GET)
+        .expect_path(INDEX_PATH)
+        .return_status(200)
+        .create_on(&server);
+    let about = Mock::new()
+        .expect_method(GET)
+        .expect_path(ABOUT_PATH)
+        .return_status(200)
+        .create_on(&server);
+
+    let mut config = common::build_configuration(&server);
+    config.no_stats = false;
+    // Start users in .5 seconds.
+    config.users = Some(2);
+    config.hatch_rate = 4;
+    config.status_codes = true;
+
+    let mut paths_and_weights = vec![(INDEX_PATH, 9), (ABOUT_PATH, 3)];
```

Comment:
Btw - that can be follow up, but we seem to never check these numbers -- only that the statistics match the actual test.

Reply:
We should open an issue: there's a todo item in the back of my mind to validate each and every configuration option is working as intended. (I am most likely to tackle this when exposing setters/getters)

Reply:
Oh - I see now where we do that just with the 1/3 pattern ...